### PR TITLE
Update VisaCheckoutSDK to 6.2

### PR DESCRIPTION
**Pull request by demerino**
Fix for https://github.com/braintree/braintree-ios-visa-checkout/issues/4

Upgrade directions from Visa docs:

**Native apps upgrading to 6.2 from a previous version**
- Delete TrustDefender.framework from your project.
- Delete VisaCheckoutButton.swift or VisaCheckoutButton .h and .m source files from your project (depending on if you have a Swift or Objective-C app).
- Delete VisaCheckoutSDK.framework from your project.
- Drag the new VisaCheckoutSDK.framework into your project's Embedded Binaries section.
- Remove any existing VisaCheckoutSDK.configure() call (probably in your AppDelegate).
- If you're using an instance of VisaCheckoutButton, use our new checkoutButton.onCheckout(profile:purchaseInfo:presenting:completion) method instead of one of the old onCheckout methods.
- If you're manually invoking Visa Checkout, use our new VisaCheckoutSDK.configureManualCheckoutSession(profile:purchaseInfo:presenting:onReady:result) method instead of the old VisaCheckoutSDK.checkout() method.
- If you referenced a VisaCheckoutButton in a Storyboard or Xib file, you will need to give the button height and width constraints as well now (assuming you're using Auto Layout).
- You may also need to clear out and reset the UIView's Custom Class for the button in Storyboard. Just set it back to UIView then reset it back to VisaCheckoutButton.

### CODE REVIEW

File: Demo/Features/Visa Checkout/BraintreeDemoVisaCheckoutViewController.m [hunk 1]
```diff
@@ -27,30 +27,44 @@ - (void)viewDidLoad {
...
+                    self.progressBlock([NSString stringWithFormat:@"Error tokenizing Visa Checkout card: %@", error.localizedDescription]);
+                } else if (tokenizedVisaCheckoutCard) {
+                    self.completionBlock(tokenizedVisaCheckoutCard);
+                } else {
+                    self.progressBlock(@"User canceled.");
+                }
+            }];
         }];
+
+//        [checkoutButton onCheckoutWithPurchaseInfo:purchaseInfo completion:^(VisaCheckoutResult *result) {
```
Comment (skunkworks) [inline Demo/Features/Visa Checkout/BraintreeDemoVisaCheckoutViewController.m]: Dead code?

**Review by skunkworks — APPROVED** [review]
Other than the commented out code, looks good.

File: Demo/Features/Visa Checkout/BraintreeDemoVisaCheckoutViewController.m [hunk 2]
```diff
@@ -27,30 +27,44 @@ - (void)viewDidLoad {
...
-        NSLog(@"Tokenizing VisaCheckoutResult...");
-        [self.client tokenizeVisaCheckoutResult:result completion:^(BTVisaCheckoutCardNonce * _Nullable tokenizedVisaCheckoutCard, NSError * _Nullable error) {
-            if (error) {
-                self.progressBlock([NSString stringWithFormat:@"Error tokenizing Visa Checkout card: %@", error.localizedDescription]);
-            } else if (tokenizedVisaCheckoutCard) {
-                self.completionBlock(tokenizedVisaCheckoutCard);
-            } else {
-                self.progressBlock(@"User canceled.");
-            }
+        [checkoutButton onCheckoutWithProfile:profile purchaseInfo:purchaseInfo presentingViewController:self completion:^(VisaCheckoutResult * _Nonnull result) {
```
Comment (quinnjn) [inline Demo/Features/Visa Checkout/BraintreeDemoVisaCheckoutViewController.m]: We'll have to update docs for this

File: UnitTests/BTVisaCheckout_Tests.swift [hunk 3]
```diff
@@ -73,7 +73,7 @@ class BTVisaCheckout_Tests: XCTestCase {
             XCTAssertEqual(visaProfile.datalevel, DataLevel.full)
             XCTAssertEqual(visaProfile.clientId, "clientExternalId")
             if let acceptedCardBrands = visaProfile.acceptedCardBrands {
-                XCTAssertEqual(acceptedCardBrands, [CardBrand.visa, CardBrand.mastercard, CardBrand.amex, CardBrand.discover])
+                 XCTAssertTrue(acceptedCardBrands.count == 4)
```
Comment (quinnjn) [inline UnitTests/BTVisaCheckout_Tests.swift]: Got some whitespace

**Review by quinnjn — APPROVED** [review]

**Pull request merged**